src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -15,7 +15,8 @@ const CPM_ABILITIES = [
     spell: SPELLS.PRAYER_OF_MENDING_CAST,
     name: 'Prayer of Mending',
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: (haste, combatant) => (12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) / (1 + haste),
+    getCooldown: (haste, combatant) => ((12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) + 1.5) / (1 + haste), // +1.5 for base cast time
+    recommendedCastEfficiency: 0.70, // There's a lot of things that can potentially delay PoM: movement, heavy damage, etc. It is important to maintain high uptime but delaying a few GCDs is not bad.
```

Comment:
Actually the default is 80%, anything higher it won't be shown. 80-75% = minor, 75% - 65% = average and <65% is major.

https://github.com/MartijnHols/WoWAnalyzer/blob/cd8901cf834fc2e7470310a753c6aad50f913af6/src/Parser/HolyPriest/CombatLogParser.js#L179